code/src/components/content/TaskFormCheckbox.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable react/jsx-closing-bracket-location */
+import React from 'react';
+import cat from 'reducers/cat';
+import tasks from 'reducers/tasks';
+import { useDispatch, useSelector } from 'react-redux';
+
+const TaskFormCheckbox = ({ task }) => {
+  const dispatch = useDispatch();
+  const taskListLength = useSelector((store) => store.tasks.tasks).length;
+  return (
+    <input
+      id={task.id}
+      onChange={() => {
+        // Changes cat to 'amazing' if task gets completed, otherwise no change
+        if (!task.isComplete) {
+          dispatch(cat.actions.changeCatAndPrevious('amazing'));
+
+          setTimeout(() => {
+            if (taskListLength === 0) {
+              dispatch(cat.actions.changeCat('angel'));
+            } else if (taskListLength > 0 && taskListLength <= 4) {
+              dispatch(cat.actions.changeCat('stressed'));
+            } else if (taskListLength > 4 && taskListLength <= 9) {
+              dispatch(cat.actions.changeCat('dizzy'));
+            } else if (taskListLength > 9) {
+              dispatch(cat.actions.changeCat('dead'));
```

Comment:
when I click on the checkbox, the animation of the cat changes to cat with stars eyes, but then it changes to sad eyes even i click on all checkboxes